utils/enums/SpidLevels.ts

```diff
@@ -0,0 +1,23 @@
+export enum SpidLevel {
+  L1 = "https://www.spid.gov.it/SpidL1",
+  L2 = "https://www.spid.gov.it/SpidL2",
+  L3 = "https://www.spid.gov.it/SpidL3"
+}
+
+type SpidComparison = Record<SpidLevel, ReadonlySet<SpidLevel>>;
```

Comment:
`ReadOnlyNonEmptySet` should be a more accurate type but only for the name.

```suggestion
import { ReadOnlyNonEmptySet } from "@pagopa/ts-commons/lib/types";

export enum SpidLevel {
  L1 = "https://www.spid.gov.it/SpidL1",
  L2 = "https://www.spid.gov.it/SpidL2",
  L3 = "https://www.spid.gov.it/SpidL3"
}

type SpidComparison = Record<SpidLevel, ReadOnlyNonEmptySet<SpidLevel>>;
```